scripts/delete_packer_vm.py

```
@@ -0,0 +1,64 @@
+    parsed_url = urlparse(proxmox_api_url)
+
+    try:
+        proxmox = ProxmoxAPI(
+            parsed_url.hostname,
+            user=f"{user}@{realm}",
+            token_name=token_name,
+            token_value=os.getenv("PROXMOX_TOKEN"),
+            verify_ssl=False,
+            port=parsed_url.port,
+        )
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)